lib/leadfoot_web/live/suspension_calculator_live/view.html.heex

```diff
@@ -1,81 +1,104 @@
-<h1 class="text-6xl font-racing-sans">LEADFOOT</h1>
-<h2 class="text-2xl font-ibm-plex-mono">Suspension Calculator</h2>
-<div class="mt-3 grid grid-cols-4 md:grid-cols-2 gap-2 text-black">
-  <div class="rounded-lg bg-slate-300 p-2">
-    <.simple_form for={@form} id="form" phx-change="validate" phx-submit="save">
-      <div class="grid-auto-fit">
-        <.input class="w-full" field={@form[:mass]} label="Mass" inputmode="numeric"/>
-        <.input
-          class="w-full"
-          field={@form[:front_distribution]}
-          type="text"
-          label="Front Distribution"
-          inputmode="numeric"
-        />
-
-        <div class="sm-grid-auto-fit">
-          <div class="inline-block">
-            <.input
-              class="w-full"
-              field={@form[:front_spring_rate]}
-              type="text"
-              label="Front Spring Rate"
-              inputmode="numeric"
-            />
-          </div>
+<div class="grid grid-cols-2 md:grid-cols-2 gap-2 text-black">
+  <div class="rounded-lg bg-transparent p-2">
+    <h1 class="text-4xl md:text-6xl font-racing-sans-one font-weight: lighter">LEADFOOT</h1>
+    <h2 class="text-1xl md:text-2xl font-ibm-plex-sans whitespace-nowrap">
+      Suspension Calculator
+    </h2>
+  </div>
+</div>
+<div class="relative rounded-lg bg-transparent p-2">
+  <LeadfootWeb.CoreComponents.modal id="create_modal">
```

Comment:
`LeadfootWeb.CoreComponents` is imported into the LiveView so you can just use `<.modal id="create_modal">`